cmd/helm/repo_remove_test.go

```diff
@@ -95,4 +84,79 @@ func TestRepoRemove(t *testing.T) {
 	if f.Has(testRepoName) {
 		t.Errorf("%s was not successfully removed from repositories list", testRepoName)
 	}
+
+	// Test removal of multiple repos in one go
+	var testRepoNames = []string{"foo", "bar", "baz"}
+	idxs := make(map[string][]string, len(testRepoNames))
+
+	// Add test repos
+	for _, repoName := range testRepoNames {
+		o := &repoAddOptions{
+			name:     repoName,
+			url:      ts.URL(),
+			repoFile: repoFile,
+		}
+
+		if err := o.run(os.Stderr); err != nil {
+			t.Error(err)
+		}
+
+		cacheIndex, cacheChart := indexing(rootDir, repoName)
+		idxs[repoName] = []string{cacheIndex, cacheChart}
+
+	}
+
+	// Create repo remove command
+	multiRmOpts := repoRemoveOptions{
+		names:     testRepoNames,
+		repoFile:  repoFile,
+		repoCache: rootDir,
+	}
+
+	// Reset the buffer before running repo remove
+	b.Reset()
+
+	// Run repo remove command
+	if err := multiRmOpts.run(b); err != nil {
+		t.Errorf("Error removing list of repos from repositories: %q", testRepoNames)
+	}
+
+	// Check that stuff were removed
+	if !strings.Contains(b.String(), "has been removed") {
+		t.Errorf("Unexpected output: %s", b.String())
+	}
+
+	for _, repoName := range testRepoNames {
+		f, err := repo.LoadFile(repoFile)
+		if err != nil {
+			t.Error(err)
+		}
+		if f.Has(repoName) {
+			t.Errorf("%s was not successfully removed from repositories list", repoName)
+		}
+		cacheIndex := idxs[repoName][0]
+		cacheChart := idxs[repoName][1]
+		testIndices(t, cacheIndex, cacheChart, repoName)
+	}
+}
+
+func indexing(rootDir string, repoName string) (cacheIndexFile string, cacheChartsFile string) {
```

Comment:
This function name does not make sense to me. Can we name it something that better indicates that it is just creating two files?

Reply:
Since it was brought up, I agree, might as well give it a descriptive name.
So, one last change please @lindhe and then we merge.